store/sqlstore/supplier.go

```diff
@@ -214,25 +214,25 @@ func (s *SqlSupplier) Next() store.LayeredStoreSupplier {
 func setupConnection(con_type string, dataSource string, settings *model.SqlSettings) *gorp.DbMap {
 	db, err := dbsql.Open(*settings.DriverName, dataSource)
 	if err != nil {
-		mlog.Critical(fmt.Sprintf("Failed to open SQL connection to err:%v", err.Error()))
+		mlog.Critical("Failed to open SQL connection to err:%v", mlog.Err(err))
```

Comment:
Here we no longer needed the '%v', probably we can just use the string "Failed to open SQL connection", and the `mlog.Err(err))`

Reply:
done